### PR TITLE
Updated namespace `Views.Repositories.Codes` to `Views.Repositories.Code`

### PR DESCRIPTION
**Pull request by DeveloperWOW64**
### ⏱️ Before you start

- [x] Have you checked if a similar PR has already been requested? 

### ↗️ Related/Fixed issues

- Related: #224 

### 📸 Assets (if appropriate):

_None Yet_

### CODE REVIEW

**Comment by onein528** [comment]
Don't rename it. many link will be broken.

**Comment by DeveloperWOW64** [comment]
> Don't rename it. many link will be broken.

sure

**Comment by onein528** [comment]
If you have ANY tiny fixes, such as margin/padding/grammer mistakes/. or improvements, you can update it.

**Comment by DeveloperWOW64** [comment]
> If you have ANY tiny fixes, such as margin/padding/grammer mistakes/. or improvements, you can update it.

sure. BTW @onein528 there's no fluenthub icon in the top left. why is this?

**Comment by onein528** [comment]
> > If you have ANY tiny fixes, such as margin/padding/grammer mistakes/. or improvements, you can update it.
> 
> sure. BTW @onein528 there's no fluenthub icon in the top left. why is this?

Store only issue. idk but I will fix that GU17. GU16 is yours!

**Comment by onein528** [comment]
Do not change like that...

File: README.md [hunk 1]
```diff
@@ -148,18 +133,38 @@ Initialize submodules recursively:
...
 Copyright (c) 2022 FluentHub Team
 
-Licensed under the MIT license stated in [LICENSE](LICENSE).
+Licensed under the MIT license stated in the [LICENSE](LICENSE).
+
+---
+
+<a href="https://github.com/FluentHub/FluentHub/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=FluentHub/FluentHub" />
+</a>
```
Comment (onein528) [inline README.md:170]: Please do not change like that. Some of those are good. but others is literally bad. Can I revert bad one?

**Review by onein528 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DeveloperWOW64) [reply README.md:170]: What's wrong?

Comment (DeveloperWOW64) [reply README.md:170]: @onein528

Comment (DeveloperWOW64) [reply README.md:170]: You want it in a different position?

Comment (DeveloperWOW64) [reply README.md:170]: > Please do not change like that. Some of those are good. but others is literally bad. Can I revert bad one?

Sure! @onein528

Comment (DeveloperWOW64) [reply README.md:170]: Ah @onein528 You want me to?

Comment (onein528) [reply README.md:170]: No, I will do that.

Comment (DeveloperWOW64) [reply README.md:170]: @onein528 Anything else you want me to change?

**Comment by DeveloperWOW64** [comment]
okay. @onein528 There's a feature I want to implement... GitHub mostly uses people's Login name instead of their name:
Login:
image
Name:
image
I want FluentHub to use name instead of login if possible. problem is that there is currently no query for Actor.Name. Could you please query it for issue, pr, discussion, etc.?

**Comment by onein528** [comment]
near to impossible. some of queries cannot be get Name which is included in User model.

**Comment by onein528** [comment]
But always *try*. I will take a look at the things what I can do.

**Comment by DeveloperWOW64** [comment]
I cannot work on this PR any longer; I have no idea what to implement.

**Comment by onein528** [comment]
ok.

**Comment by DeveloperWOW64** [comment]
@onein528 Please review.

**Comment by onein528** [comment]
I have a situation. So, not gonna happen today, sorry.

**Review by onein528 — CHANGES_REQUESTED** [review]
There’re a lot of things that need to be changed. I will take a look what should be changed.

**Comment by DeveloperWOW64** [comment]
> There’re a lot of things that need to be changed. I will take a look what should be changed.

@onein528?

File: README.md [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
 
+<br/>
+
 ---
 
-## Installation
+## ➕ Installation
 
-### Via Microsoft Store
+##### Via Microsoft Store
```
Comment (onein528) [inline README.md:29]: Only phrases and emoji additions are allowed among the changed README.md diff.

- Not allowed:
  - Add `##` before header 3nd.

**Review by onein528 — CHANGES_REQUESTED** [review]
And also need to restore Beta logo sets

File: README.md (hunk 2 above)
Comment (DeveloperWOW64) [reply README.md]: Finished!

**Comment by onein528** [comment]
Can you @DeveloperWOW64 revert 7f145c45b903ee49a0e154da1e80189e299cf148 and 54df4f512db4665d8cffa70d3d193a7d027d4214 ?

**Comment by DeveloperWOW64** [comment]
@onein528 Changed all the things you requested. 😁

**Comment by onein528** [comment]
Great. Processing successfully, I will merge this!

**Comment by DeveloperWOW64** [comment]
@onein528 AZP succeeded! Now merge?

**Review by onein528 — APPROVED** [review]
Looks good to me.

**Comment by onein528** [comment]
@DeveloperWOW64 Thank you!

**Pull request merged**